cBioPortal/cbioportal-frontend
# generic assay continuous data study view implementation

Fix cBioPortal/cbioportal#7657

Describe changes proposed in this pull request:
- **New add chart tabs**
![image](https://user-images.githubusercontent.com/15748980/102393775-cfe5d880-3fa6-11eb-862b-334803e1b55b.png)

![image](https://user-images.githubusercontent.com/15748980/102393829-e2f8a880-3fa6-11eb-9cd8-cebe15007438.png)

![image](https://user-images.githubusercontent.com/15748980/100655640-93b14780-3319-11eb-9717-18a679ea502e.png)

- **Increased width of Add chart tooltip (50px)**
previous: 
![image](https://user-images.githubusercontent.com/15748980/100656006-10dcbc80-331a-11eb-8188-a54e53f27ae8.png)

after: 
![image](https://user-images.githubusercontent.com/15748980/100655941-fe628300-3319-11eb-85fb-5a553d986f97.png)


--------------------------**Testing**--------------------------
Multiple studies (contains 2 `GENERIC ASSAY TYPE` data: `Microbiome Signature` and `Treatment Response`)
https://deploy-preview-3521--cbioportalfrontend.netlify.app/study/summary?id=brca_tcga_pan_can_atlas_2018%2Clgg_tcga_pan_can_atlas_2018%2Cccle_broad_2019

tests can be performed:
1. Click `Add Chart`, and click the arrow, then select `Microbiome Signature` or `Treatment Response` to add a new chart
2. After add a chart, select a range in the new added chart see if selected samples updated correctly
3. download chart data or go to comparison page from the selected chart
4. refresh the page and see if added charts still in place
